spec/lib/authorities/linked_data/find_term_spec.rb

```diff
@@ -218,9 +220,13 @@
         end
 
         context 'when alternate authority name is used to access loc' do
-          let(:results) { lod_loc.find('sh 85118553', subauth: 'subjects') }
+          before do
+            stub_request(:get, 'http://id.loc.gov/authorities/subjects/sh85118553')
+              .to_return(status: 200, body: webmock_fixture('lod_loc_term_found.rdf.xml'), headers: { 'Content-Type' => 'application/rdf+xml' })
+            allow(lod_loc.term_config).to receive(:authority_name).and_return('ALT_LOC_AUTHORITY')
```

Comment:
Had to adjust this test to add the stub here.  The test broke during an auto-merge by github.